api/src/resources/source.rs

```diff
@@ -26,6 +27,8 @@ pub struct Source {
 
     #[serde(rename = "_kind")]
     pub kind: SourceKind,
+    #[serde(default, rename = "email_transform_tag")]
+    pub transform_tag: Option<TransformTag>,
```

Comment:
Maybe just rename this field to `email_transform_tag` on these objects, since that's what you called it on sources anyway.